packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -684,4 +684,59 @@ describe('DomainForm', () => {
         expect(form).toMatchSnapshot();
         form.unmount();
     });
+
+    test('using allowImportExport', () => {
+        const domain = DomainDesign.create({});
+
+        const form = mount(
+            <DomainForm
+                domain={domain}
+                onChange={jest.fn()}
+                allowImportExport={true}
+            />
+        );
+
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
+        expect(form.find('.domain-form-manual-section').length).toEqual(1);
```

Comment:
This is a duplicate from the line above (line 699).

Reply:
Ah yup, an accident, removed!